modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/web/internal/layout/display/page/test/JournalArticleLayoutDisplayPageProviderTest.java

```diff
@@ -69,6 +75,48 @@ public void tearDown() throws Exception {
 		ServiceContextThreadLocal.popServiceContext();
 	}
 
+	@Test
+	public void testGetLayoutDisplayPageObjectProviderJournalArticleWithExpiredArticleVersionInfoItemReference()
+		throws Exception {
+
+		ServiceContext serviceContext =
+			ServiceContextTestUtil.getServiceContext(_group.getGroupId());
+
+		JournalArticle originalArticle = JournalTestUtil.addArticle(
+			_group.getGroupId(),
+			JournalFolderConstants.DEFAULT_PARENT_FOLDER_ID);
+
+		JournalArticleLocalServiceUtil.updateArticle(
+			originalArticle.getUserId(), originalArticle.getGroupId(),
+			originalArticle.getFolderId(), originalArticle.getArticleId(),
+			originalArticle.getVersion(), originalArticle.getTitleMap(),
+			originalArticle.getDescriptionMap(), originalArticle.getContent(),
+			originalArticle.getLayoutUuid(), serviceContext);
+
+		originalArticle = JournalArticleLocalServiceUtil.updateStatus(
```

Comment:
@bakayattila, can't we use _journalArticleLocalService.expireArticle here? I know that it calls updateStatus as you used it, but it would be more clearer

Reply:
I can use that if it is preferred.